NAMESPACE

```diff
@@ -9,6 +9,7 @@ export(orderly_bundle_list)
 export(orderly_bundle_pack)
 export(orderly_bundle_pack_remote)
 export(orderly_bundle_run)
+export(orderly_cancel_remote)
```

Comment:
should this be `orderly_remote_cancel` to mirror `orderly_remote_status` or stay `orderly_cancel_remote` to mirror `orderly_run_remote`? Naming things is hard

Reply:
We have more functions which are `orderly_x_remote`, though they do tend to be when there is a local equivalent - which there isn't here. I think `orderly_cancel_remote` feels more natural wording to me so I am happy to stick with that, I don't mind switching it over if you prefer though